tests/rptest/tests/offset_for_leader_epoch_archival_test.py

```diff
@@ -122,3 +133,91 @@ def wait_for_topic():
                 # and we can't read from cloud storage.
                 assert epoch_end_offset != -1, f"{epoch_end_offset} vs -1"
                 assert epoch_end_offset >= offset, f"{epoch_end_offset} vs {offset}"
+
+    def num_manifests_uploaded(self):
+        s = self.redpanda.metric_sum(
+            metric_name=
+            "redpanda_cloud_storage_spillover_manifest_uploads_total",
+            metrics_endpoint=MetricsEndpoint.PUBLIC_METRICS)
+        self.logger.info(
+            f"redpanda_cloud_storage_spillover_manifest_uploads = {s}")
+        return s
+
+    def produce(self, topic, msg_count):
+        msg_size = 1024 * 256
+        topic_name = topic.name
+        producer = KgoVerifierProducer(self.test_context,
+                                       self.redpanda,
+                                       topic_name,
+                                       msg_size=msg_size,
+                                       msg_count=msg_count)
+
+        producer.start()
+        producer.wait()
+        producer.free()
+
+        def all_partitions_spilled():
+            return self.num_manifests_uploaded() > 0
+
+        wait_until(all_partitions_spilled, timeout_sec=180, backoff_sec=10)
+
+    @cluster(num_nodes=4, log_allow_list=RESTART_LOG_ALLOW_LIST)
+    def test_querying_archive(self):
+        topic = TopicSpec(redpanda_remote_read=True,
+                          redpanda_remote_write=True)
+        epoch_offsets = {}
+
+        self.client().create_topic(topic)
```

Comment:
nit: This resolves to call into `KafkCliTools.create_topic`, which doesn't use the remote r/w topic configs from line 166. The topic has remote r/w enabled from the cluster configs in `SISettings`, so you could just remove the `redpanda_remote_read|write` args above.